doc/Doxyfile

```diff
@@ -107,7 +107,7 @@ BRIEF_MEMBER_DESC      = YES
 # brief descriptions will be completely suppressed.
 # The default value is: YES.
 
-REPEAT_BRIEF           = YES
+REPEAT_BRIEF           = NO
```

Comment:
These two changes conflicts with the WIP on the sphinx/doxygen harmonization, and doesn't have anything to do with documenting `cr_assume_*` macros.